syntax = "proto3";
import "google/protobuf/any.proto";

message Apply_data
{
    string apply_name = 1;
    uint64 apply_time =2;
}

message Apply_list
{
    string apply_list_add = 1;
    uint32 apply_list_id = 2;
}

message Test
{
    string name = 1;
    map<string, google.protobuf.Any> data = 2;
}
// protoc --cpp_out=./ *.proto
